frontend/common/constants.ts

```diff
@@ -466,6 +472,7 @@ export default {
       'Set different values for your feature based on what segments users are in.',
     TAGS_DESCRIPTION:
       'Organise your flags with tags, tagging your features as "<strong>protected</strong>" will prevent them from accidentally being deleted.',
+    TOOLTIP_METADATA_DESCRIPTION: 'Add metadata in your',
```

Comment:
Doesn't look complete

Reply:
It is not incomplete, it is used like this `tooltip={${Constants.strings.TOOLTIP_METADATA_DESCRIPTION} segments}`